idea/common/src/enums/api-methods.enum.ts

```diff
@@ -1,6 +1,9 @@
 enum API_METHODS {
   PROGRAM_DATA = 'program.data',
   PROGRAM_META_ADD = 'program.meta.add',
+  PROGRAM_STATE_ADD = 'program.state.add',
```

Comment:
We should be able to get metadata and state wasms for particular code too.

Reply:
Done